app/src/main/res/values-nb/strings-nb.xml

```diff
@@ -1,501 +1,830 @@
+    <string name="navigation_drawer_close">Lukk navigasjonsmenyen</string>
+    <string name="title_activity_calibration_check_in">Kalibreringsaktivitet</string>
+    <string name="title_activity_usb_connected">USB-tilkoblet aktivitet</string>
+    <string name="snooze">Utsett</string>
+    <string name="snooze_hint">Utsett varsling i x minutter</string>
+    <string name="ok">OK</string>
+    <string name="cancel">Avbryt</string>
+    <string name="no">Nei</string>
+    <string name="pref_sensor_location_dont_ask_again"> Ikke spør meg igjen</string>
+
+    <string name="title_activity_error">"Feil- og Handlingslist"</string>
+    <string name="title_activity_main">Hovedaktivitet</string>
+    <string name="title_activity_fake_numbers">Falske Nummer</string>
+    <string name="title_activity_edit_alert">Rediger Alarm</string>
+    <string name="title_activity_double_calibration">Dobbel Kalibrering</string>
+    <string name="title_activity_calibration_override">Overstyr Kalibrering</string>
+    <string name="pref_header_cloud_storage">Skylagring</string>
+    <string name="auto_configure_title">Autokonfigurer</string>
+    <string name="prefs_auto_config_summary">Autokonfigurer med strekkode.</string>
+    <string name="scan_share2_barcode">Skan Delingsstrekkode</string>
+    <string name="pref_share2_scan_barcode_summary">Eller skanne strekkoden på mottakeren</string>
+
+    <!-- MongoDB Settings -->
+    <string name="pref_title_mongodb">MongoDB</string>
+    <string name="pref_summary_mongodb_enabled">Hvis du bruker MongoLab og Azure dette må være aktivert</string>
+    <string name="pref_title_mongodb_uri">URI</string>
+    <string name="pref_dialog_mongodb_uri">Skriv MongoDB URI"</string>
+    <string name="pref_message_mongodb_uri">Erstatt eksempelverdier i {}ene med dine riktige verdier</string>
+    <string name="pref_default_mongodb_uri">mongodb://{user}:{password}@{host}.mongolab.com:{11111}/{database}</string>
+    <string name="pref_title_mongodb_collection">Samlingsnavn</string>
+    <string name="pref_dialog_mongodb_collection">Skriv inn navnet på samlingen</string>
+    <string name="pref_message_mongodb_collection">Dette er navnet på samlingen hvor CGM data vil bli lagret</string>
+
+    <string name="pref_title_mongodb_device_status_collection">Samlingsnavn for enhetsstatus</string>
+    <string name="pref_dialog_mongodb_device_status_collection">Skriv inn samlingsnavnet for Enhetsstatus"</string>
+    <string name="pref_message_mongodb_device_status_collection">Dette er navnet på samlingen der batteriet og andre enhetsdata vil bli lagret</string>
+
+    <!-- API Settings -->
+    <string name="pref_title_api">Nightscout synkronisering (REST)</string>
+    <string name="pref_title_api_enabled">Aktivert</string>
+    <string name="pref_summary_api_enabled">REST API er et alternativ til direkte mongodb opplasting</string>
+    <string name="pref_title_api_url">Grunn-URL</string>
```

Comment:
Base-URL would be the correct form here. "Grunn" seems incorrect

Reply:
Done